src/transformers/pipelines/image_feature_extraction.py

```diff
@@ -1,12 +1,14 @@
 from typing import Dict
 
-from ..utils import add_end_docstrings, is_vision_available
+from ..utils import add_end_docstrings, is_vision_available, is_torch_available
```

Comment:
@aliencaocao The quality check is failing here because of the order of the imports, you can see this from the message [on the CI](https://app.circleci.com/pipelines/github/huggingface/transformers/95234/workflows/55ed01d0-3d25-4be5-9dc7-9dee8c55ecbf/jobs/1252624) 

I'd strongly recommend running in a virtualenv where running ruff locally is possible. In this case, there's just a single change to make, but it quickly becomes very difficult to contribute to the library if you have to touch many files or there's lots of places the format doesn't match the library's expected format. 

```suggestion
from ..utils import add_end_docstrings, is_torch_available, is_vision_available
```

Reply:
Thanks and I will definitely setup one.